x-pack/plugins/maps/common/elasticsearch_util/elasticsearch_geo_utils.ts

```diff
@@ -381,14 +381,7 @@ export function createSpatialFilterWithGeometry({
   relation: ES_SPATIAL_RELATIONS;
 }): GeoFilter {
   ensureGeoField(geoFieldType);
```

Comment:
Nit: Can you just remove geoFieldType parameter from method. Not really needed anymore.